ibis/pandas/tests/test_operations.py

```diff
@@ -178,6 +179,23 @@ def test_timestamp_functions(case_func, expected_func):
     assert ibis.pandas.execute(result) == expected
 
 
+@pytest.mark.skipif(PY2, reason="not enabled on PY2")
+def test_times_ops(t, df):
+    result = t.plain_datetimes_naive.time().between('10:00', '10:00').execute()
+    expected = np.zeros(len(df), dtype=bool)
+    tm.assert_numpy_array_equal(result, expected)
+
+    result = t.plain_datetimes_naive.time().between('01:00', '02:00').execute()
+    expected = np.ones(len(df), dtype=bool)
+    tm.assert_numpy_array_equal(result, expected)
+
+
+@pytest.mark.skipif(not PY2, reason="testing for PY2")
+def test_times_ops_py2(t, df):
+    with pytest.raises(ValueError):
+        t.plain_datetimes_naive.time().between('10:00', '10:00').execute()
```

Comment:
Can you put only the part that fails underneath `raises` so that it's clear when reading the test where the code should fail.